architecture/infrastructure_components/web_console.adoc

```diff
@@ -78,3 +78,50 @@ as the link:../../dev_guide/quota.html[quota] and
 link:../../dev_guide/limits.html[resource limits] that are set on your project.
 <8> Visualization of the contents of your project.
 ====
+
+== Java Components Console
```

Comment:
Maybe "Integration Components Console"?  The JMX view is a small part, bulk of the stuff is in the Camel and ActiveMQ views...